.github/workflows/pr.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
         os: [ubuntu-latest, macos-latest, windows-latest]
         java: [11, 17, 21]
         include:
-          - os: flyci-macos-large-latest-m1
+          - os: macos-latest
```

Comment:
`macos-latest` is already included in the above matrix so this seems to duplicate it.

We might consider using `macos-13` since `-latest` is now aarch64 and `-13` is still x86...